packages/web-app-text-editor/src/App.vue

```diff
@@ -231,12 +231,19 @@ export default {
   }
 }
 </script>
-<style lang="scss" scoped>
+<style lang="scss">
 #text-editor-preview {
   max-height: 80vh;
   overflow-y: scroll;
 }
 #text-editor-input {
   resize: vertical;
 }
+.editor-wrapper-height {
+  height: calc(100% - 42px);
+}
+
+#text-editor-input {
```

Comment:
Why not use the `oc-height-1-1` for this element also and remove the custom style rule? :)

Reply:
if the height is set 100% and the text is longer than the height of the textarea (scrollbar showing), the upper buttons div gets stagely cut by me, the only workaround I found was to subtract the height of the upper buttons
![Screenshot from 2022-05-16 12-40-49](https://user-images.githubusercontent.com/7430156/168575862-29bcd2df-c49a-4e74-ac17-c5f97fc0a828.png)

Reply:
If I set the class, it sets the div around <textarea> to 100% but not  <textarea> itself. So the class passing would need to be implemented in ocTextarea for this purpose.